modules/custom/openy_search/openy_google_search/openy_google_search.install

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @file
+ * Module installation file.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function openy_google_search_install() {
+
+  if (\Drupal::service('path.alias_storage')->aliasExists('/search', 'en')) {
```

Comment:
Not critical, but it would be nice to define services in separate variables and add PHPDoc notation, thus IDE autocomplete will be available and you can easily find a class or what's happened in a method of the service.
For example:
```
    /** @var \Drupal\Core\Path\AliasStorageInterface $path_alias_storage */
    $path_alias_storage = \Drupal::service('path.alias_storage');
```
It would be nice to use this approach for all services in the code.